example/BPI/Testnet/Setup.hs

```diff
@@ -70,7 +73,11 @@ runSetup workDir = do
   setLocaleEncoding utf8
   createRequiredDirs
   sockPath <- getEnv "CARDANO_NODE_SOCKET_PATH"
-  let nodeInfo = BPI.NodeInfo (Testnet $ NetworkMagic 2) sockPath
+  let nodeInfo =
+        LocalNodeConnectInfo
+          (CardanoModeParams $ EpochSlots (60 * 60 * 24))
+          (Testnet $ NetworkMagic 2)
+          sockPath
```

Comment:
Are these changes necessary because of upstream BPI changes? I kind of missed them

Reply:
Yes, `BPI.queryProtocolParams` is no longer a thing and you need this to run query effect